src/nodejsWorker.ts

```diff
@@ -1,9 +1,38 @@
 var logPrefix = "LanguageWorkerConsoleLog";
+var ACTIVE_LTS_VERSION = "v8";
+var CURRENT_BRANCH_VERSION = "v10";
 var worker;
+
+// Node version validation
+try {

```

Comment:
Move this to a method and tests with valid versions and invalid semver versions.